assets/data/themes/drauven.json

```diff
@@ -316,13 +316,19 @@
 				{
 					"name": "Drauv_ArmR",
 					"tint": "skin", "tintAmount": 0.4,
-					"depth": 1200,
+					"depth": 1350,
 					"rigMode": "oneHanded"
 				},
 				{
-					"name": "Drauv_ArmR_2handed",
+					"name": "Drauv_ArmR_2handed_lower",
```

Comment:
Since this `name` has changed, we should probably also rename the `warriorF_Drauv_ArmR_2handed`, `hunterM_Drauv_ArmR_2handed`, `hunterF_Drauv_ArmR_2handed`, `mysticM_Drauv_ArmR_2handed` and `mysticF_Drauv_ArmR_2handed` files so that those models don't lose a limb when they try to hold a 2-handed item 🙂 
(Just appending either `_lower` or `_upper` to the existing filenames should be enough)